src/3133R0_first_factor.md

```diff
@@ -344,10 +362,20 @@ library meaningfully harder to implement.  It would also mean that other project
 integer factorization would be unable to take advantage of the work, even though it's already done
 and included (although hidden) in their standard library.
 
+Even that may not be enough.  It's possible that even the ideal implementation will run into
+compiler iteration limits for certain inputs.  If that turns out to be true, then declining to add
+this function means that we can't have a standard units library, either --- at least, not one that
+supports every unit.
+
 # Conclusion
 
 Integer factorization is perhaps the most basic and useful operation in all of number theory.  While
 basic implementations are easy to write, robust and efficient implementations are tricky and subtle.
-Providing a high quality implementation in the C++ standard library would unblock at least a future
-standard units and quantities library.  Given the wide diversity of use cases for integer
+If results are required at compile time, the difficulty increases even more, because compilers limit
+the number of algorithmic iterations.  In fact, there's a real risk that a `constexpr` function that
+produces results for every 64-bit input is even possible for end user code!
```

Comment:
Did you mean?

```suggestion
produces results for every 64-bit input is even impossible for end user code!
```

Reply:
I meant "not even possible".  Nice catch!